java/ql/src/Security/CWE/CWE-023/PartialPathTraversalFromRemote.qhelp

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<include src="PartialPathTraversalOverview.inc.qhelp"> 
```

Comment:
Include a see-also note here like the one in the other qhelp file

Reply:
I don't understand what you're saying here, sorry

Reply:
One qhelp file contains `<p>See also <code>java/partial-path-traversal-from-remote</code>, which is similar to this query but only flags instances with evidence of remote exploitability</p>`

The other one should have a similar note pointing in the opposite direction.